Sources/Rendering/Misc/FullScreenRenderWindow/index.js

```diff
@@ -94,8 +94,14 @@ function vtkFullScreenRenderWindow(publicAPI, model) {
   model.interactor.initialize();
   model.interactor.bindEvents(model.container);
 
+  const superClass = { ...publicAPI };
+
   // Expose background
-  publicAPI.setBackground = model.renderer.setBackground;
+  publicAPI.setBackground = (background) => {
+    // Synchronize the setting of background from here to renderer
+    model.renderer.setBackground(background);
+    superClass.setBackground(background);
+  };
 
```

Comment:
`publicAPI.setBackground()` call line 147 should probably be moved here or at the end of this function